cluster/mesos/docker/mesos-slave/bin/wrapdocker

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+
+# Copyright 2013-2014 Jérôme Petazzoni, Johan Haleby, lalyos, James Harris,
+#                     Michael Elsdörfer, Tony Hesjevik, Esben Haabendal,
+#                     Michael A. Smith
+# Copyright 2015 The Kubernetes Authors All rights reserved.
```

Comment:
If these were only being used locally by the deploy scripts, I'd agree. However, in this case they're injected into docker images and expected to be on the PATH. It would be pretty strange in that case...